Session-4/CurrencyExchangeApp/CurrencyExchangeApp/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using CurrencyExchange;
+using System;
+using System.Linq;
+
+namespace CurrencyExchangeApp
+{
+   internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            if (args.Count() > 0)
+            {
+                if (args[0].Count<char>() > 3 || args[1].Count<char>() > 3)
+                {
+                    Console.WriteLine("Enter a valid Currency[Ex : USD, INR , AUD]");
+                }
+                else
```

Comment:
Remove else block and add return statement inside if condition to avoid deep nesting.